crates/rust-analyzer/src/config.rs

```diff
@@ -6,7 +6,7 @@
 //! Of particular interest is the `feature_flags` hash map: while other fields
 //! configure the server itself, feature flags are passed into analysis, and
 //! tweak things like automatic insertion of `()` in completions.
-
+#![allow(dead_code)]
```

Comment:
This should be removed ideally

Reply:
there are parts that actually belong to what is to be implemented, so I can't remove this just yet.